src/cr/cube/matrix.py

```diff
@@ -260,14 +260,17 @@ def _columns_inserted_at_right(self):
     def _iter_columns(self):
         """Generate all column vectors with insertions interleaved at right spot."""
         opposing_insertions = self._all_inserted_rows
+        column_vector_index = 0
```

Comment:
This was the first thing I saw, and I had to think: "what is the column vector index?". Then I looked above to see what the name of the method was, and then I _remembered_ that I originally wrote `i`. Maybe change it to just `column_index`, since the name of the method implies that (and not `_iter_column_vectors`).